securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -1,5 +1,10 @@
 import pytest
-import urllib2
+
+try:
+    from urllib.request import urlopen, Request
+except ImportError:  # Python 2/3 compatibility
+    from urllib2 import urlopen, Request
```

Comment:
it can, but i think we should leave this as is because when we excise six everywhere when we dump support for Python 2 it's easier to just remove the python 2 import here (this code was the recommendation from https://python-future.org/compatible_idioms.html)

Reply:
Fair